docs/README.md

```diff
@@ -4721,8 +4721,152 @@ curl -i GET \
 
 ```
 
+>**Sample response body**
+
+```
+{
+              "@odata.context": "/redfish/v1/$metadata#NetworkAdapterCollection.NetworkAdapterCollection",
+              "@odata.id": "/redfish/v1/Chassis/a022faa5-107c-496d-874e-89c9f3e2df1c:1/NetworkAdapters",
+              "@odata.type": "#NetworkAdapterCollection.NetworkAdapterCollection",
+              "Description": "The collection of network adapter resource instances available in this chassis.",
+              "Members": [{
+                             "@odata.id": "/redfish/v1/Chassis/a022faa5-107c-496d-874e-89c9f3e2df1c:1/NetworkAdapters/{rid}"
+              }],
+              "Members@odata.count": 1,
+              "Name": "NetworkAdapterCollection",
+              "Oem": {
+                             "Hpe": {
+                                           "@odata.context": "/redfish/v1/$metadata#HpeNetworkAdapterStatus.HpeNetworkAdapterStatus",
+                                           "@odata.type": "#HpeNetworkAdapterStatus.v1_0_0.HpeNetworkAdapterStatus",
+                                           "MemberContents": "AllDevices"
+                             }
+              }
+}
+
+
+```
+
+## Single  Network Adapter
```

Comment:
There is an extra space between the words.
Also, change "Single" to "Individual"